cmake/grpc_generate_cpp.cmake

```diff
@@ -0,0 +1,104 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A helper function to generate C++ code from given .proto files.
+cmake_minimum_required(VERSION 3.8)
+find_package(Threads REQUIRED)
+
+function(grpc_generate_cpp SRCS HDRS INCLUDE_DIRS)
```

Comment:
This will be helpful for reuse in other proposed HoloHub apps, thank you!

Could you please add a short header to clarity expected inputs:
- `SRCS`, `HDRS`, `INCLUDE_DIRS` values should be set to the names of list targets to which respective paths will be added
- `.proto` files to be listed as trailing arguments